PittAPI/lab.py

```diff
@@ -19,7 +19,6 @@
 
 import requests
 from bs4 import BeautifulSoup
-
```

Comment:
Normally I wouldn't impose my style on the project, but I already had this committed and this stuck around after rebasing...